ath.sh

```diff
@@ -3,40 +3,40 @@ set -o errexit
 set -o nounset
 set -o pipefail
 set -o xtrace
-cd $(dirname $0)
+cd "$(dirname "$0")"
```

Comment:
```
In ath.sh line 6:
cd $(dirname $0)
   ^-----------^ SC2046 (warning): Quote this to prevent word splitting.
             ^-- SC2086 (info): Double quote to prevent globbing and word splitting.
```